packages/block-editor/src/store/selectors.js

```diff
@@ -1914,7 +1914,9 @@ export const __experimentalGetParsedPattern = createSelector(
 		}
 		return {
 			...pattern,
-			blocks: parse( pattern.content ),
+			blocks: parse( pattern.content, {
+				__unstableSkipMigrationLogs: true,
```

Comment:
I wonder if it should be just a "verbose" stable option like a lot of tools offer. Anyway, I agree with this change as well. 

That said, I wonder if and how we can surface these for pattern authors somehow.

Reply:
I didn't use something generic like `verbose` because there could be other logs as well - ex. `invalid block` logs, but no strong opinions here.

> That said, I wonder if and how we can surface these for pattern authors somehow.

Yeah, I'm not sure what would be a good solution here and it seems that this is something that pattern authors should have in mind to check at times. The root problem for me though is that we don't have a minimum supported WP version for each pattern and this should be looked into when we implement that. What I mean is that even if a deprecation is handled, it's not so easy to decide to update the pattern because it might cause invalidation of it in older WP versions. Some prior art was [this PR](https://github.com/WordPress/wordpress-develop/pull/1483).

Reply:
I've spent a lot of time in the parser code and wondered myself why we issue all those logs. something like `debug()`'s interface would be a boost where we can opt-in to messages coming from `wp.blocks.api.parser.migration`

I'm in favor of turning off the logs for everyone unless requested, but I also don't rely on those messages so I'm sure I'm overlooking part of the value they bring

Reply:
> I didn't use something generic like `verbose` because there could be other logs as well - ex. `invalid block` logs, but no strong opinions here.

Agreed. And the same goes for a hypothetical numeric `verbosity` option: we don't have a stable model to classify log messages consistently, so introducing either option today would lead to a confusing interface, IMO. A plain `__unstableSkipMigrationLogs` seems like the way to go.